### PR TITLE
coreos-ostree-importer: various improvements; add requester script

### PR DESCRIPTION
**Pull request by dustymabe**
There are various improvements to the ostree importer in this patchset, including
the addition of send-ostree-import-request.py. Please see individual commit
messages for more information.

### CODE REVIEW

File: coreos-ostree-importer/README.md [hunk 1]
```diff
@@ -99,3 +99,24 @@ You'll have to update the body with new information you'd like to use. Then run:
 ```
 ./publisher.py
 ```
+
+## Fedora Messaging sender from the Fedora CoreOS Pipeline
+
+Included in this directory is a file (`fcos-pipeline-ostree-import-request.py`)
```
Comment (jlebon) [inline coreos-ostree-importer/README.md]: Minor: maybe something a little more concise, e.g. `send-ostree-import-request.py`? The fact that it's used in the FCOS pipeline is a detail I think.

File: coreos-ostree-importer/README.md [hunk 2]
```diff
@@ -99,3 +99,24 @@ You'll have to update the body with new information you'd like to use. Then run:
...
+
+Here's how you might send a request using `fcos-pipeline-ostree-import-request.py`: 
+
+
+```
+cosa buildprep --build=31.20200212.20.0 s3://fcos-builds/prod/streams/testing-devel/builds
+/usr/lib/coreos-assembler/fcos-pipeline-ostree-import-request.py \
+        --fedmsg-conf /srv/fedora-messaging-config.toml \
+        --build 31.20200212.20.0 --stg \
+        --s3 fcos-builds/prod/streams/testing-devel \
```
Comment (jlebon) [inline coreos-ostree-importer/README.md]: This is actually written down by `buildprep` now: https://github.com/coreos/coreos-assembler/blob/5b8ff1fe481b125bc7965084eb345c95fd439ca1/src/cmd-buildprep#L53-L55. But yeah, I think still it's better to be explicit.

File: coreos-ostree-importer/fcos-pipeline-ostree-import-request.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/python3
+
+"""
+    This script is meant to be run from the Fedora CoreOS build
+    pipeline (see https://github.com/coreos/fedora-coreos-pipeline.git)
+    It makes an ostree import request to the coreos-ostree-importer
```
Comment (jlebon) [inline coreos-ostree-importer/fcos-pipeline-ostree-import-request.py]: Minor: "an OSTree import request" :)

**Comment by dustymabe** [comment]
all comments addressed .. still need to test this in a dev pipeline run

**Pull request merged**